src/app/components/server-compare-charts/server-compare-charts.component.html

```
@@ -927,90 +720,25 @@ <h3 class="text-xl font-semibold flex gap-2 items-center">
-                  >
-                    {{ chartItem.show_more ? "Hide" : "Show" }} Details
-                  </button>
-                </div>
-              }
             </td>
           </tr>
           @if (chartItem.show_more) {
-            @for (chartData of chartItem.data; track chartData) {
-              @if (chartItem.data.length > 1) {
+            @for (chartData of chartItem.data ?? []; track chartData) {
+              @if ((chartItem.data?.length ?? 0) > 1) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)